### PR TITLE
DOC-7888 rename role

### PR DESCRIPTION
**Pull request by osfameron**

### CODE REVIEW

**Review by RichardSmedley — APPROVED** [review]
:+1:

**Review by maria-robobug — APPROVED** [review]
LGTM 👍

File: modules/howtos/pages/provisioning-cluster-resources.adoc [hunk 1]
```diff
@@ -128,10 +128,10 @@ include::howtos:example$provisioning_resources_collections.py[tag=drop_collectio
 include::howtos:example$provisioning_resources_collections.py[tag=drop_scope]
 ----
 
-Note that the most minimal permissions to create and drop a Scope or Collection is
-https://docs.couchbase.com/server/7.0/learn/security/roles.html#manage-collections-in-scope[Scope Admin]
+Note that the most minimal permissions to create and drop a Scope or Collections is
```
Comment (maria-robobug) [inline modules/howtos/pages/provisioning-cluster-resources.adoc]: ah sorry, missed this, `Collection` instead of `Collections`.

Comment (osfameron) [reply modules/howtos/pages/provisioning-cluster-resources.adoc]: Hah, I was sure you'd pulled me up on this already, and indeed it is in the dotnet version I was using as template, so correcting there too...

Comment (maria-robobug) [reply modules/howtos/pages/provisioning-cluster-resources.adoc]: Ah, reminds me I should actually look at the .NET PR too, I'd totally forgotten sorry.

**Pull request merged**